src/grammar-parser/grammar-parser.jison

```diff
@@ -222,6 +223,9 @@ number
   | number '%' {
       $$ = $1 * 0.01;
     }
+  | NUMBER 'e' NUMBER {
+      $$ = yy.evaluateByOperator('*', [$1, yy.evaluateByOperator('^', [10, $3])])
+    }
```

Comment:
decimal_number: 
(the first three cases)

number:
- decimal_number
- decimal_number e decimal_number
- decimal_number %